### PR TITLE
Script to install/update rbe into the istio-testing project

### PR DESCRIPTION
**Pull request by fejta**
ref https://github.com/istio/test-infra/pull/1400 #1396 

`bazel --config=remote --remote_instance_name=projects/istio-testing/instances/default_instance` now available to anyone with access to the istio-testing project.

### CODE REVIEW

File: rbe/install-istio-testing.sh [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+if [[ ! -f configure.sh ]]; then
+  echo "ERROR: configure.sh not found" >&1
+  echo "  curl https://raw.githubusercontent.com/kubernetes/test-infra/master/rbe/configure.sh -o configure.sh" >&1
+  exit 1
+fi
+
+
+proj=istio-testing
+pool=prow-pool
```
Comment (utka) [inline rbe/install-istio-testing.sh:19]: should this node be created first, or it is part of the install ?

Comment (fejta) [reply rbe/install-istio-testing.sh:19]: The script creates an rbe pool with this name

**Pull request merged**